index.html

```diff
@@ -184,62 +185,96 @@
 
 - TFS supports Git and mixing Git/TFVC repos in the same project.
 - Git is the default version control provider for new projects in Azure DevOps (Online & On-Prem).
-- Microsoft advises using Git over TFVC.
 - Integrated Git support in Visual Studio and Visual Studio Code.
   - You will want to learn some basic commands.
+- Comparison table showing that Git can do everything that TFVC can.
+  - [Comparison of Git and TFVC capabilities](https://docs.microsoft.com/en-us/azure/devops/repos/tfvc/comparison-git-tfvc?view=azure-devops)
+---
+
+## Migrating to Git
 
-- [Comparison of Git and TFVC capabilities](https://docs.microsoft.com/en-us/azure/devops/repos/tfvc/comparison-git-tfvc?view=azure-devops)
+### Keep your history in the past!
+
+- Microsoft suggests doing **tip migrations**.
+  - Bring only a snapshot of the most recent version of a branch into Git.
+--
+
+- Why?
+  - Lossy process, at best.
+  - Possibly misleading.
+  - High level of effort involved.
+  - Relative rarity usage of the history.
+--
+
+- Where *do* you keep the history then?
+  - In your old version control system!
+  - Refer to your old repositories in your README.md files.
+---
+
+## Migrating to Git
+
+### Keep your history in the past!
+
+Quote from Microsoft's [Plan your migration to Git - Migrating history](https://docs.microsoft.com/en-us/azure/devops/learn/git/centralized-to-git#migrating-history):
+
+> "For most development teams, the time spent trying to migrate history is typically better spent on other areas of the migration that have a higher return on investment, especially improving processes. [...] In almost all cases, we recommend the tip migration."
```

Comment:
Formatting with MD + paragraphs, one sentence per line, will auto format to paragraph.
```suggestion
> "For most development teams, the time spent trying to migrate history is typically better spent on other areas of the migration that have a higher return on investment, especially improving processes. 
> [...] In almost all cases, we recommend the tip migration."
```

Reply:
Are you suggesting that we break up the single-lined, continuous paragraph into multiple lines? That would be for the readability of the MD source?

Reply:
yeah, it's really just affecting the readability of the md source.